### ISSUE TITLE
Dreambooth training

### ISSUE THREAD

**Issue by kdziedzic66** [ShivamShrirao/diffusers#15]
### Describe the bug

Hello,

I'd like to ask a question about the training of dreambooth on smaller GPUs: it is reported in README, that 9.82gb should be enogh to run the training with int8 adam, gradients checkpointing enabled and mixed_precision set to fp16. 

We are trying ro run the training and it suceeds only if we downscale the resolution to 256 (512 is default i belive). Are those reported values for the 256 resolution and it has not been specified ? We are working with NVIDIA GeForce RTX 2080 Ti 11264MiB.

### System Info

- `diffusers` version: 0.4.0.dev0
- Platform: Linux-5.4.0-125-generic-x86_64-with-glibc2.31
- Python version: 3.9.14
- PyTorch version (GPU?): 1.12.1+cu102 (True)
- Huggingface_hub version: 0.10.0
- Transformers version: 4.22.2
- Using GPU in script?: <fill in>
- Using distributed or parallel set-up in script?: <fill in>

**Comment by omarkazmi** [comment]
I got it working at 512x512 on linux (lubuntu) with my 2080ti, but I had to free up ALL my resources by booting into the console with no window manager. The desktop environment pulls 100 Mb of VRAM, and it turns out dreambooth really needed that last 100 Mb.

**Comment by ShivamShrirao** [comment]
Yeah, those are numbers for 512x512 resolution. Have you installed my fork of diffusers ?
`pip install git+https://github.com/ShivamShrirao/diffusers.git` and was xformers successfully compiled and installed for your system ?

**Comment by kdziedzic66** [comment]
Installation of xformers has helped. Thank you very much!
